Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -56,8 +63,9 @@ function InternalInit
     $repositoryIdGuid = [Guid]::Parse($repositoryId);
```

Comment:
Could use Guid.TryParse instead and provide a user friendly message instead of a guid parse exception with no real context or actionable error.